.github/workflows/publish.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish
```

Comment:
This workflow can either be renamed to "Publish docs" or be combined with the `publish_to_pypi.yml` file

Reply:
Renamed to Publish docs